projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -248,11 +253,12 @@ private function prepare_menu_item( array $menu_item ) {
 		}
 
 		$item = array(
-			'icon'  => $this->prepare_menu_item_icon( $menu_item[6] ),
-			'slug'  => sanitize_title_with_dashes( $menu_item[2] ),
-			'title' => $menu_item[0],
-			'type'  => 'menu-item',
-			'url'   => $this->prepare_menu_item_url( $url, $parent_slug ),
+			'icon'       => $this->prepare_menu_item_icon( $menu_item[6] ),
+			'slug'       => sanitize_title_with_dashes( $menu_item[2] ),
+			'title'      => $menu_item[0],
+			'type'       => 'menu-item',
+			'url'        => $this->prepare_menu_item_url( $url, $parent_slug ),
+			'identifier' => $this->prepare_menu_item_identifier( $menu_item[5] ),
```

Comment:
It would be great if `$menu_item[5]` could be a named var so that it's more self-documenting.